src/main/kotlin/com/sourcegraph/cody/config/SettingsMigration.kt

```diff
@@ -96,18 +96,18 @@ class SettingsMigration : Activity {
         .filter { it.llm == null }
         .forEach {
           val (model, provider, title) =
-              modelToProviderAndTitle.getOrDefault(it.model, Triple("", "Default", "Default"))
+              modelToProviderAndTitle.getOrDefault(it.model, Triple(it.model, null, null))
           val llmState = LLMState()
-          llmState.provider = provider
-          llmState.title = title
           llmState.model = model
+          llmState.title = title
+          llmState.provider = provider
           it.llm = llmState
-          it.model = null
         }
   }
 
   private val modelToProviderAndTitle =
       mapOf(
+          "Claude 2 by Anthropic" to Triple("anthropic/claude-2", "Anthropic", "claude 2"),
```

Comment:
For enterprise the model request looks like this:
```
[Trace - 03:55:48 PM] Sending request 'chat/models - (22)'
Params: {
  "id": "553a1e2f-d73d-48a6-a962-2c7271cd6db4"
}


[Trace - 03:55:48 PM] Received response 'chat/models - (22)' in 2ms
Result: {
  "models": [
    {
      "default": true,
      "codyProOnly": false,
      "provider": "Anthropic",
      "title": "claude 2",
      "model": "anthropic/claude-2"
    }
  ]
}
Error: null
```

Reply:
This line fixes the issue with the failing migration and "Unknown by Unknown" display name